Makefile

```diff
@@ -28,6 +28,8 @@ GO_LDFLAGS ?= $(COMMIT_FLAG) $(VERSION_FLAG)
 GOOS ?= $(shell $(GO) version | sed 's/^.*\ \([a-z0-9]*\)\/\([a-z0-9]*\)/\1/')
 GOARCH ?= $(shell $(GO) version | sed 's/^.*\ \([a-z0-9]*\)\/\([a-z0-9]*\)/\2/')
 
+CGO_ENABLED ?= 0
```

Comment:
```suggestion
# allow overriding CGO_ENABLED for scenarios where gomplate must be compiled with CGO enabled, such as when using boringcrypto.
CGO_ENABLED ?= 0
```

Reply:
Updated as suggested. Marking this conversation as resolved.